src/pages/test-report/components/competency/__tests__/competency.spec.ts

```diff
@@ -85,38 +81,170 @@ describe('CompetencyComponent', () => {
         expect(hammerProvider.addPressAndHoldEvent).toHaveBeenCalledWith(component.recordFault);
       });
     });
+
     describe('getLabel', () => {
       it('should get the correct label for a competency', () => {
         component.competency = Competencies.controlsSteering;
         expect(component.getLabel()).toBe('Steering');
       });
     });
+
     describe('recordFault', () => {
       it('should dispatch a ADD_DRIVING_FAULT action', () => {
         component.competency = Competencies.controlsSteering;
 
+        const storeDispatchSpy = spyOn(store$, 'dispatch');
```

Comment:
no need to assign the result of spyOn to storeDispatchSpy, you can just call expect(store$.dispatch).toHaveBeenCalled
multiple places

Reply:
You are right, however being a bit more explicit always improves readability.